gui/src/lib/api.ts

```diff
@@ -287,10 +287,13 @@ export const api = (() => {
         const { workspaces } = (await invoke('describe-workspaces', {})) as any;
         return workspaces;
       },
-      async createProject(root: string, templateUrl?: string): Promise<string> {
+      async createProject(
+        root: string,
+        templateUrl: string | null,
+      ): Promise<string> {
         const { id } = (await invoke('create-project', {
           root,
-          templateUrl,
+          templateUrl: templateUrl !== null ? templateUrl : undefined,
```

Comment:
This is a bit messy, are you sure it isn't cleaner to use `string | undefined` instead of `string | null`?

Alternatively we can modify the backend as well but I'm not sure how it works so I didn't want to change what was actually being submitted to the server @brandonbloom

Reply:
I can change that

Reply:
Actually, nevermind, you should just be able to pass `templateUrl`. `null` is a valid (preferable?) value to send to the server here.